### PR TITLE
chore(webapi): add request duration logging

### PR DESCRIPTION
**Pull request by subotic**
subtask of #2

### CODE REVIEW

**Comment by subotic** [comment]
Hope you like it :-)

File: webapi/src/main/scala/org/knora/webapi/routing/AroundDirectives.scala [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+ *  License along with Knora.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+package org.knora.webapi.routing
+
+import akka.event.LoggingAdapter
+import akka.http.scaladsl.server.Directive0
+import akka.http.scaladsl.server.Directives._
+
+object AroundDirectives {
```
Comment (benjamingeer) [inline webapi/src/main/scala/org/knora/webapi/routing/AroundDirectives.scala]: Could you add a Scaladoc comment to this class?

Comment (subotic) [reply webapi/src/main/scala/org/knora/webapi/routing/AroundDirectives.scala]: Sure. Will do.

Comment (subotic) [reply webapi/src/main/scala/org/knora/webapi/routing/AroundDirectives.scala]: Done

**Pull request merged**